tests/integration/examples/xgboost/steps/loader/loader_step.py

```diff
@@ -43,12 +43,12 @@ def data_loader() -> (
         mode="w", delete=False, suffix=".html", encoding="utf-8"
     ) as f:
         f.write(requests.get(TRAIN_SET_RAW, timeout=13).text)
-        mat_train = xgb.DMatrix(f.name)
+        mat_train = xgb.DMatrix(f.name + "?format=libsvm")
```

Comment:
Do we want to use libsvm by default? While it's great for sparse data, it's seems pretty poor for dense data and can get quite slow.

Reply:
It's only used in this specific test, not as a default inside the integration